location-service/src/test/java/uk/gov/cshr/locationservice/service/RegionLookupServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.cshr.locationservice.service;
+
+import uk.gov.cshr.locationservice.RegionLookup;
+import uk.gov.cshr.locationservice.UK_NUTS;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+public class RegionLookupServiceTest {
+
+
+    @Test
+    public void testRegionLookups() throws Exception {
+
+        Assert.assertEquals(UK_NUTS.UKC, RegionLookup.findRegion(55.2970314, -1.72889996));
```

Comment:
Please include a test for null coordinates supplied and coordinates that would not be found in any region